package.json

```diff
@@ -23,7 +23,7 @@
     "intl": "1.2.5",
     "jail-monkey": "2.3.0",
     "jsc-android": "241213.1.0",
-    "mattermost-redux": "github:mattermost/mattermost-redux#1c2faee2660a3fdd69485ea44f0dea89b0204426",
+    "mattermost-redux": "github:mattermost/mattermost-redux#pull/964/head",
```

Comment:
this is a temporary fix, so test pass.. but before this is ready to be merged we need to update with the proper commit hash

Reply:
@enahum These will need to be merged at the same time to prevent conflicts, so perhaps we should handle updating the commit? If so, would you mind approving it and then I can handle making sure the branches are merged correctly